File: php/testcode/benchmark_test_00340.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00340(BenchmarkRequest $req): BenchmarkResponse {
    $rules = ['name' => 'required|string', 'email' => 'required|email'];
    $validated = array_intersect_key($_POST, $rules);
    $model = new stdClass();
    foreach ($validated as $k => $v) {
        $model->$k = $v;
    }
    return BenchmarkResponse::ok('validated');
}

File: rust/testcode/benchmark_test_00340.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let data = req.param("data");
    let ciphertext = secure_encrypt(&data);
    super::shared::BenchmarkResponse::ok(&ciphertext)
}

fn secure_encrypt(data: &str) -> String {
    let iv = [42u8; 12];
    format!("AES256GCM:{}:iv_included", data.len())
}
